code/HubPage.php

```diff
@@ -33,6 +33,32 @@ public function getCMSFields()
     {
         $fields = parent::getCMSFields();
         $fields->removeByName('Content');
+
+        // Display the Taxonomy and Type as a single selectable item
+        //TODO more elegant way would be to get a PR in taxonomy module to provide a concatenated name and then use this in the gridfield.
+        $components = GridFieldConfig_RelationEditor::create();
+        $components->removeComponentsByType('GridFieldAddNewButton');
+        $components->removeComponentsByType('GridFieldEditButton');
+
+        $autoCompleter = $components->getComponentByType('GridFieldAddExistingAutocompleter');
+        $autoCompleter->setResultsFormat('$Name ($TaxonomyType)');
+
+        $dataColumns = $components->getComponentByType('GridFieldDataColumns');
+        $dataColumns->setDisplayFields(array(
+            'Name' => 'Term',
+            'TaxonomyType' => 'Type'
+        ));
+
+        $fields->addFieldToTab(
+            'Root.Tags',
+            GridField::create(
+                'Terms',
+                'Terms',
```

Comment:
Why is terms appearing twice (line 55 and 56)

Reply:
one is the ID the other is the Label - GridField is a bit funny like that... see http://api.silverstripe.org/3/GridField.html#method___construct

Reply:
ping @mandy-kate any other issues you can see of am I good to merge? :)

Reply:
Hi @camfindlay sorry just looked at this now, looks good to go thanks :+1:

Reply:
🍍 👍 :shipit: